apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/VedtaksbrevService.kt

```diff
@@ -60,9 +59,26 @@ class VedtaksbrevService(
         }
     }
 
+    fun ferdigstillVedtaksbrev(behandlingId: String): Boolean {
+        logger.info("Ferdigstiller vedtaksbrev for behandlingId=$behandlingId")
+
+        val vedtaksbrev = hentVedtaksbrev(behandlingId)
+            ?: throw IllegalArgumentException("Vedtaksbrev ikke funnet. Avbryter ferdigstilling/attestering.")
+
+        return if (vedtaksbrev.status in listOf(Status.OPPRETTET, Status.OPPDATERT)) {
```

Comment:
Hvordan forholder Brev-status seg til Behandling-status? Er dette muligens to sider av samme sak?

Reply:
Akkurat nå er det ingen kobling mellom de, men det er definitivt noe vi må se på før prod.